packages/vite/src/node/server/transformRequest.ts

```diff
@@ -73,7 +73,7 @@ export function transformRequest(
   const pending = server._pendingRequests.get(cacheKey)
   if (pending) {
     return server.moduleGraph
-      .getModuleByUrl(removeTimestampQuery(url), options.ssr)
+      .getModuleByUrl(url, options.ssr)
```

Comment:
Is this removal intentional.

Reply:
yes, see https://github.com/vitejs/vite/pull/13231#discussion_r1196460956

Reply:
I added it back, I can send another PR later, better to avoid the noise